### PR TITLE
add subject;fix html/text; self templates folder

### PR DESCRIPTION
**Pull request by lc4t**
This change is [Reviewable](https://reviewable.io/reviews/andyfangdz/django-asyncmailer/6)

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/andyfangdz/django-asyncmailer/pull/6" target="_blank">ReviewNinja

**Comment by andyfangdz** [comment]
2 Style checks failed. Did not `tox` before pushing? See travis log.

File: asyncmailer/views.py [hunk 1]
```diff
@@ -73,31 +71,41 @@ def get_json(request):
...
+        emails = json.load(request.FILES['upload'])['email']
+    if formats == 'text':
+        for email in emails:
+            async_select_and_send(email, subject, str(payload))
+    else:
+        for email in emails:
+            async_mail([email], subject, context_dict=payload,
+                       template=template.replace('.html', '-templates/')
+                       + template)
+    return HttpResponse('send success')
```
Comment (andyfangdz) [inline asyncmailer/views.py]: Can you change this into something more formal? Like json `{"success": true}`

**Comment by andyfangdz** [comment]
Good job! Fix the issues and let's merge it in.

**Pull request merged**